bin/pd/Makefile

```diff
@@ -33,6 +33,7 @@ ckanapi := $(REGISTRY_CKANAPI_COMMAND)
 ogc_search := $(OGC_SEARCH_COMMAND)
 oc_search := $(OC_SEARCH_COMMAND)
 backup_dir := $(PD_BACKUP_DIRECTORY)
+tmp_dir := $(PD_TMP_DIRECTORY)
```

Comment:
I think we can default this one to $TMPDIR, the normal system default, so we don't have to update our dev environments with a setting only needed in prod